packages/salesforcedx-vscode-core/src/orgPicker/orgList.ts

```diff
@@ -50,64 +48,58 @@ export class OrgList implements vscode.Disposable {
     }
   }
 
-  public async getAuthInfoObjects() {
-    const authFilesArray = await AuthInfo.listAllAuthorizations().catch(
-      err => null
-    );
+  public async getOrgAuthorizations(): Promise<OrgAuthorization[]> {
+    const orgAuthorizations = await AuthInfo.listAllAuthorizations();
+    return orgAuthorizations;
+  }
 
-    if (authFilesArray === null || authFilesArray.length === 0) {
-      return null;
-    }
-    const authInfoObjects: FileInfo[] = [];
-    for (const authFile of authFilesArray) {
-      try {
-        const filePath = path.join(
-          await ConfigFile.resolveRootFolder(true),
-          '.sfdx',
-          authFile.username + '.json'
-        );
-        const fileData = readFileSync(filePath, 'utf8');
-        authInfoObjects.push(JSON.parse(fileData));
-      } catch (e) {
-        console.log(e);
-      }
-    }
-    return authInfoObjects;
+  public async getAuthFieldsFor(username: string): Promise<AuthFields> {
+    const authInfo: AuthInfo = await AuthInfo.create({
+      username
+    });
+    return authInfo?.getFields();
```

Comment:
I tested locally it doesn't look like the ? is needed here.  Super weird how the create method is doc'd so I see how got there, but I think it can be removed.